crates/cli-flags/src/lib.rs

```diff
@@ -89,7 +89,7 @@ wasmtime_option_group! {
         /// Whether or not to enable parallel compilation of modules.
         pub parallel_compilation: Option<bool>,
         /// Whether to enable proof-carrying code (PCC)-based validation.
-        pub enable_pcc: Option<bool>,
+        pub pcc: Option<bool>,
```

Comment:
I should highlight this as a drive-by change by me (cc @cfallin), but this'll rename `-C enable-pcc` to `-C pcc` (most other options omit the "enable" or "disable" to have everything be "if you specify it it's enabled unless you say `=n` in which case you're asking to disable it")